### PR TITLE
feat: Add documentation for customer current usage

### PR DESCRIPTION
**Pull request by vincent-pochet**
Describe new `GET /api/v1/customers/:customer_id/current_usage` API end point to retrieve the current usage of a customer

### CODE REVIEW

File: docs/api/12_customer_usage/custome-usage.mdx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Get customer current usage
+
+## Route
+
+```curl title="GET"
+/api/v1/customers/:customer_id/current_usage
```
Comment (mathieu-lago) [inline docs/api/12_customer_usage/custome-usage.mdx:13]: Do we actually need `:` before `customer_id`?

File: docs/api/12_customer_usage/custome-usage.mdx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+  </TabItem>
+
+  <TabItem value="ruby" label="Ruby">
+
+  ```ruby
+  require 'lago-ruby-client'
+
+  client = Lago::Api::Client.new({api_key: '__YOUR_API_KEY__'})
+  customer_usage = client.customer.current_usage('')
```
Comment (mathieu-lago) [inline docs/api/12_customer_usage/custome-usage.mdx:63]: Should it be `('')` or `('customer_id')` at the end?

File: docs/api/12_customer_usage/customer-usage-object.mdx [hunk 3]
```diff
@@ -0,0 +1,115 @@
+---
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# The customer usage object
+
+This object represents the charge usage of a customer of your business on the current billing period.<br/>
```
Comment (mathieu-lago) [inline docs/api/12_customer_usage/customer-usage-object.mdx:10]: "This object represents the usage-based charges associated with one of your customers for the current billing period."

File: docs/api/12_customer_usage/customer-usage-object.mdx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+It lets you monitor the customer usage
```
Comment (mathieu-lago) [inline docs/api/12_customer_usage/customer-usage-object.mdx:11]: "It allows you to monitor customer usage throughout the period."

File: docs/api/12_customer_usage/customer-usage-object.mdx [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    "from_date": "2022-07-01",
+    "to_date": "2022-07-31",
+    "issuing_date": "2022-08-01",
+    "amount_cents": 123,
+    "amount_currency": "EUR",
+    "total_amount_cents": 123,
+    "total_amount_currency": "EUR",
+    "vat_amount_cents": 0,
+    "vat_amount_currency": "EUR",
+    "charges_usage": [
```
Comment (mathieu-lago) [inline docs/api/12_customer_usage/customer-usage-object.mdx:27]: `{` instead of `[`?

File: docs/api/12_customer_usage/customer-usage-object.mdx [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+| **amount_cents** &nbsp &nbsp <Type>Integer</Type> &nbsp &nbsp <NotNullable>Not null</NotNullable> | Amount in cents, VAT (tax) excluded|
+| **amount_currency** &nbsp &nbsp <Type>String</Type>&nbsp &nbsp <NotNullable>Not null</NotNullable> | Currency of the amount |
+| **charge** &nbsp &nbsp <Type>JSON</Type> &nbsp &nbsp <NotNullable>Not null</NotNullable> | Related [charge](#charge) object |
+| **billable_metric** &nbsp &nbsp <Type>JSON</Type> &nbsp &nbsp <NotNullable>Not null</NotNullable> | Related [billable metric](#billable-metric-object) object |
+
+## Charge object
+
+| Attributes | Description |
+| -----------| ----------- |
+| **lago_id** &nbsp &nbsp <Type>String</Type> &nbsp &nbsp <NotNullable>Not null</NotNullable> | ID of the charge in lago application |
```
Comment (mathieu-lago) [inline docs/api/12_customer_usage/customer-usage-object.mdx:75]: "in the Lago application"

File: docs/api/12_customer_usage/customer-usage-object.mdx [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+| **charge_model** &nbsp &nbsp <Type>String</Type> &nbsp &nbsp <NotNullable>Not null</NotNullable> | Model of pricing applied by the charge.<br/>See [charge documentation](/docs/guide/plans/charges#the-charge-models) for possible values |
+
+
+## Billable Metric object
+
+| Attributes | Description |
+| -----------| ----------- |
+| **lago_id** &nbsp &nbsp <Type>String</Type> &nbsp &nbsp <NotNullable>Not null</NotNullable> | ID of the billable metric in lago application |
```
Comment (mathieu-lago) [inline docs/api/12_customer_usage/customer-usage-object.mdx:83]: "in the Lago application"

File: docs/api/12_customer_usage/custome-usage.mdx (hunk 1 above)
Comment (vincent-pochet) [reply docs/api/12_customer_usage/custome-usage.mdx:13]: Yes, it's a standard way to show that this `customer_id` has to be replaced by a real customer id

File: docs/api/12_customer_usage/custome-usage.mdx (hunk 2 above)
Comment (vincent-pochet) [reply docs/api/12_customer_usage/custome-usage.mdx:63]: `('customer_id')`, good catch!

File: docs/api/12_customer_usage/customer-usage-object.mdx (hunk 5 above)
Comment (vincent-pochet) [reply docs/api/12_customer_usage/customer-usage-object.mdx:27]: No, it's an array of objects

File: docs/api/12_customer_usage/customer-usage-object.mdx [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+          "charge_model": "graduated"
+        },
+        "billable_metric": {
+          "lago_id": "99a6094e-199b-4101-896a-54e927ce7bd7",
+          "name": "Usage metric",
+          "code": "usage_metric",
+          "aggregation_type": "sum"
+        }
+      }
+    ]
```
[the remark this answers is not in the document]
Comment (vincent-pochet) [reply docs/api/12_customer_usage/customer-usage-object.mdx:43]: Same, it's the closing tag for the array of object

**Pull request merged**